### PR TITLE
New update for old and useful project

### PR DESCRIPTION
**Pull request by GreatBahram**
Hi
First of all thank you for your useful module. And I think it's better to have a function for converting Persian numbers to English numbers. Secondly I added that to your project and update the license to GPL3 and plus that I update the `readme` file. 
Finally I increment the beta version to `0.0.4` and please update this updates to `pypi` too.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Python Version of [Persian.js](https://github.com/itmard/persian.js)
 
 ###Install Package
 ```
-pip install persian
+pip3 install persian
```
Comment (itmard) [inline README.md]: Why changed this ? the official tool name is pip (specially using a virtualenv ) people can change it to other aliases like pip3

**Comment by GreatBahram** [comment]
:-D But I believe GNU GPL is better than MIT. 
Please check it now, I changed it as you said.

**Comment by itmard** [comment]
Please check changes in the project function names and update your mr
Also I changed license to Apache license 2.0

**Comment by GreatBahram** [comment]
Hmmm. Nice update. I changed my code as you said. I was wondering why you change the license ?

**Comment by itmard** [comment]
@GreatBahram Thanks for this update.

**Pull request merged**